src/System.CommandLine/Binding/ArgumentConverter.DefaultValues.cs

```diff
@@ -19,7 +19,8 @@ internal static partial class ArgumentConverter
                   .SingleOrDefault(c => c.GetParameters().Length == 0)!);
 #endif
 
-    private static Array CreateEmptyArray(Type itemType, int capacity = 0)
+    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL3050", Justification = "We are working on it")]
+    private static Array CreateArray(Type itemType, int capacity = 0)
```

Comment:
Can we drop the default ‘0’ value? If you want a 0 length array use Array.Empty

Reply:
I've done that in the place where it was possible. Since it's used mostly from non-generic context we can't use `Array.Empty<T>`

Reply:
```suggestion
    private static Array CreateArray(Type itemType, int capacity)
```

I don't see any calls to this method that use the default value.